pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -21,10 +23,12 @@ import (
 const (
 	// DIDExchange did exchange protocol
 	DIDExchange        = "didexchange"
-	connectionSpec     = metadata.AriesCommunityDID + ";spec/connections/1.0/"
+	connectionSpec     = metadata.AriesCommunityDID + ";spec/didexchange/1.0/"
 	connectionInvite   = connectionSpec + "invitation"
 	connectionRequest  = connectionSpec + "request"
 	connectionResponse = connectionSpec + "response"
+	//TODO : : Acknowledgement needs to follow RFCS-0015 for acks : https://github.com/hyperledger/aries-rfcs/tree/master/features/0015-acks
```

Comment:
@talwinder50 what's this TODO for?

Reply:
Based on above comments this doesnt apply anymore
https://github.com/hyperledger/aries-framework-go/pull/183#discussion_r320472909